app/package.json

```diff
@@ -7,8 +7,9 @@
     },
     "dependencies": {
         "base-64": "^0.1.0",
+        "@expo/vector-icons": "^8.1.0",
```

Comment:
How does this compare to [react-native-vector-icons](https://www.npmjs.com/package/react-native-vector-icons)?